includes/blocks/course-theme/class-lesson-actions.php

```diff
@@ -161,6 +161,21 @@ public function render( array $attributes = [] ): string {
 
 		$course_id = Sensei()->lesson->get_course_id( $lesson_id );
 
+		$is_learning_mode  = \Sensei_Course_Theme_Option::has_learning_mode_enabled( $course_id );
+		$is_awaiting_grade = false;
+
+		$lesson_status = \Sensei_Utils::user_lesson_status( $lesson_id, get_current_user_id() );
```

Comment:
The Psalm is marking an issue here. Could we fix the PHPDoc return of `user_lesson_status` to also have an array of `WP_Comments`, which seems to be another possible return?